onnxruntime/test/python/transformers/test_rotary_mha_fusion.py

```diff
@@ -295,23 +295,244 @@ def create_k_path_hf(self, model_type: str):
         )
         k_nodes = [reshape_k_node, transpose_k_1_node]
 
-        if model_type in {"past", "merged"}:
+        if model_type == "70b_distributed_merged":
             concat_k_node = helper.make_node(
+                    "Concat",
+                    inputs=["past_key", "k_rope"],
+                    outputs=["present_key"],
+                    axis=2,
+                )
+            k_nodes.append(concat_k_node)
+
+            shape_k1 = helper.make_node(
+                "Shape",
+                inputs=["present_value"],
+                outputs=["shape_k1_out"],
+                name="Shape_k1"
+            )
+            k_nodes.append(shape_k1)
+            shape_k2 = helper.make_node(
+                "Shape",
+                inputs=["present_value"],
+                outputs=["shape_k2_out"],
+                name="Shape_k2"
+            )
+            k_nodes.append(shape_k2)
```

Comment:
We can remove the many `k_nodes.append` calls, create a list of nodes for the 70B model, and then extend the existing `k_nodes` list with `k_nodes.extend(k_nodes_for_70b_model)`.